src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -357,47 +354,156 @@ def executeMcpTestSteps(self, config, test_type):
       self._sas_admin.InjectPeerSas({'certificateHash': certificate_hash,
                                      'url': sas_test_harness_object.getBaseUrl()})
       sas_test_harness_objects.append(sas_test_harness_object)
-
-
+   
     # Step 4,5 : Inject IAP protected entities into UUT
     for iteration_content in config['iterationData']:
-
       # Execute steps for single iteration
-      self.executeSingleMCPIteration(test_type, iteration_content, sas_test_harness_objects,
+      self.executeSingleMCPIteration(test_type, iteration_content, sas_test_harness_objects, 
                                      domain_proxy_objects)
-
+    
     # Stopping Test harness servers
     for test_harness in sas_test_harness_objects:
       test_harness.shutdown()
       del test_harness
 
-  def checkMcpIap(self, protected_entity, agg_interference_list, iap_refmodel_margin_list, delta_iap):
-    """Calculating IAP for each FSS and ESC sensor protection point,
-    and for at least 95% of the protection points of each PPA and GWPZ.
-    """
-    pass
+  def performIapAndAggregateInterferenceCheck(self, protected_entities, sas_uut_fad_object,
+                                      sas_th_fad_objects, cbsd_list):
+    """Compare the interference calculated using iap and aggregate reference models.
 
-  def callAggregateInterferenceModel(self, cbsd_list, protected_entity):
-    """Args:
-         cbsd_list: List of cbsds with Authorized grants
-         protected_entity: A dictionary with multiple key-value pairs where
-         keys are protection entities
+    Args:
+      protected_entities : List of entity records needed to be protected 
+      sas_uut_fad_object : FAD object from SAS UUT
+      sas_th_fad_objects : a list of FAD objects from SAS Test Harness
+      cbsd_list : List of DP objects registered to the SAS.
```

Comment:
Probably the var name should change to reflect the fact that this is a list of DP objects, not a list of CBSDs.

Reply:
Changed the variable name 